.yamato/config/_editor.metafile

```diff
@@ -6,4 +8,47 @@ platforms:
   - name: Win
   - name: Linux
   - name: iPhone
-agent: cds_ops_ubuntu_small
\ No newline at end of file
+editor_priming_agent: cds_ops_ubuntu_small
+
+### EDITOR PINNING
+editor_pin_agent: package_ci_ubuntu_small
+
+# remove the following once editor pinning is done. these values will be picked up from __shared
```

Comment:
I know we have these on master, but should we? (have these lines) ie shouldn't we remove these lines both here and on master?
Isn't the editor pinning finished?

Reply:
True, you have a good point. I left them there in case bigger editor pinning changes come up, so its quick to use these again, but indeed they don't add much value at this point